### PR TITLE
Polishing general UI

### PR DESCRIPTION
**Pull request by Duncid**
## Description
Polishing general UI and Sparkle, fixing UI issues

## Risk
UI breaks

### CODE REVIEW

File: front/components/assistant/conversation/SidebarMenu.tsx [hunk 1]
```diff
@@ -228,25 +228,44 @@ export function AssistantSidebarMenu({ owner }: AssistantSidebarMenuProps) {
...
+                  className="shrink"
+                  tooltip="Create a new conversation"
+                  href={`/w/${owner.sId}/assistant/new`}
+                  onClick={() => {
+                    setSidebarOpen(false);
+                    const { cId } = router.query;
+                    const isNewConversation =
+                      router.pathname === "/w/[wId]/assistant/[cId]" &&
+                      typeof cId === "string" &&
+                      cId === "new";
```
Comment (aubin-tchoi) [inline front/components/assistant/conversation/SidebarMenu.tsx:243]: `nit`: that's a bit redundant

File: front/components/assistant_builder/AssistantBuilder.tsx [hunk 2]
```diff
@@ -505,26 +529,28 @@ export default function AssistantBuilder({
...
+                  className={cn(
+                    isPreviewButtonAnimating && "animate-breathing-scale"
                   )}
                 />
               )}
               {rightPanelStatus.tab === null && template !== null && (
-                <div className="flex flex-col gap-3 rounded-full border border-structure-200 p-4">
-                  <IconButton
+                <div className="flex flex-col gap-3">
+                  <Button
```
Comment (aubin-tchoi) [inline front/components/assistant_builder/AssistantBuilder.tsx:541]: Why not keep the IconButton here and change the props accordingly (same for the other 2 occurrences)?

File: front/components/assistant/conversation/SidebarMenu.tsx (hunk 1 above)
Comment (JulesBelveze) [reply front/components/assistant/conversation/SidebarMenu.tsx:243]: Not many other options I'm afraid..

File: front/components/assistant_builder/AssistantBuilder.tsx (hunk 2 above)
Comment (Duncid) [reply front/components/assistant_builder/AssistantBuilder.tsx:541]: IconButton is deprecated (we're rolling it out).

File: front/components/assistant/conversation/SidebarMenu.tsx (hunk 1 above)
Comment (Duncid) [reply front/components/assistant/conversation/SidebarMenu.tsx:243]: That's not from the PR (pre-existing code)

File: front/components/assistant_builder/AssistantBuilder.tsx (hunk 2 above)
Comment (aubin-tchoi) [reply front/components/assistant_builder/AssistantBuilder.tsx:541]: makes sense, I'll take it

**Review by aubin-tchoi — APPROVED** [review]

**Pull request merged**